Godeps/Godeps.json

```diff
@@ -111,11 +110,6 @@
 			"Rev": "034ba8b88fdcc3d5100ae0d2edc914673b15a2f4"
 		},
 		{
-			"ImportPath": "github.com/docker/docker/pkg/parsers",
-			"Comment": "v1.4.1-165-g034ba8b",
-			"Rev": "034ba8b88fdcc3d5100ae0d2edc914673b15a2f4"
-		},
-		{
```

Comment:
These packages shouldn't be deleted, manually undo, or do the godep save from Linux.